### DISCUSSION TITLE
Change DeleteOperation#destroy to #delete

### DISCUSSION THREAD

**Discussion by paulcsmith** [luckyframework/lucky#1533] [Feature requests & Ideas] [answered]
Thoughts on changing `DeleteOperation#destroy` back to delete? I'm finding it a bit hard to remember where to use `destroy` because it is a `Delete` operation and you call `deleted?` but the method is `destroy`. I'm guessing this partly came from Rails but IMO I think we should be consistent and use `delete` everywhere so it is easier to remember and document.

Thoughts?

**Comment by jwoertink** [comment, marked as answer]
I actually debated this a bit when I made it. Ultimately I went with `destroy` just because we use `create` instead of `save`, and yeah, I was following that whole Rails naming... I do like `delete` though, and I'd be down with using that instead. We actually used to use `save`, and changed that, so easy change here too.

**Reply by paulcsmith** [reply]
Right on. I'll open an issue to change it to `delete`. I think `create` is ok though because `save` is ambiguous. In Rails it can update or create I think. I like that `create` is super clear about the action. Thoughts?

**Comment by stephendolan** [comment]
I'm the same way. I don't use Rails very much anymore and it's kind of like inserting a USB drive... I always seem to get it wrong the first time and use the wrong term 😄
